optbinning/scorecard/scorecard.py

```diff
@@ -350,7 +353,7 @@ def predict_proba(self, df):
         self._check_is_fitted()
 
         df_t = df[self.binning_process_.variable_names]
-        df_t = self.binning_process_.transform(df_t)
+        df_t = self.binning_process_.transform(df_t, metric_special = self._metric_special, metric_missing = self._metric_missing)
```

Comment:
Minor comment: remove spaces between "=". Keep maximum-length 79 to satisfy PEP-8.